src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  margin-top: 16px;
+  align-items: flex-end;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+  }
+
+  &__star-review {
+    margin: 0;
+    padding-left: 17px;
+    font-family: $font-family;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: $main-accent-color;
```

Comment:
The variable `$main-accent-color` is used here, but it's not defined in this file. Ensure that `$main-accent-color` is defined in a global SCSS file or imported into this file to avoid compilation errors.